docs/parca-agent-labelling.md

```diff
@@ -0,0 +1,69 @@
+# Labelling
+
+## Metadata labels
+
+The agent does its best to enrich profiles with labels coming from various system facilities relative to the originating process.
+
+### Profiler
+
+* `__name__`: The name of the profiler (e.g. `parca_agent_cpu`).
+* `pid`: The PID of the process.
```

Comment:
I have been wondering if it should be `__pid__`

Reply:
Yes! `__meta_pid` i would say